### ISSUE TITLE
How to host a game in a custom gametype?

### ISSUE THREAD

**Issue by giuliofoletto** [rdntcntrl/ratoa_gamecode#38]
I would like to know how to setup a server in one of the custom gametypes listed [here](https://ratmod.github.io/game-types.html). The documentation isn't really clear and explains only how to call a vote, but not how to setup the server. Are those modes associated to a `g_gametype` number?

**Comment by rdntcntrl** [comment]
Yeah the server-side documentation is unfortunately still lacking (although the OA wiki has some general info on server setup that still applies).

Some of the new gametypes have their own `g_gametype` number, namely TH (13) and Multi-Duel (14). 
The others are done through modifiers on existing gametypes. For "Extermination" (Wipeout) you would use something like:

```
set g_gametype 8
set elimination_respawn 5
set elimination_respawn_increment 5
set elimination_roundtime 0
```

Freeze can be enabled using `g_freeze 1` (also in combination with `g_gametype 8`).

**Comment by giuliofoletto** [comment]
Ok thanks. Do you need help in writing the docs? I don't know much but I might write some stuff down when I discover it and submit a PR if it is open source.

**Comment by rdntcntrl** [comment]
Help is always appreciated. The repo for https://ratmod.github.io is here: https://github.com/ratmod/ratmod.github.io
Right now it doesn't have anything server-related in it though, so unless you want to start that section from scratch it might be difficult.

I'm also not sure if we really want to write a detailed documentation of every setting for the server-side (takes a lot of time to write and maintain).
Maybe some example server configurations (for different gametypes) + a server setup tutorial would suffice?
